internal/common/voting/election.go

```diff
@@ -55,8 +55,90 @@ func (e *Election) CloseBallot() shared.ClientID {
 }
 
 func (e *Election) bordaCountResult() shared.ClientID {
-	// TODO implement Borda count winner selection method.
-	return e.pluralityResult()
+	// Implement Borda count winner selection method
+	//(may need to modify if methods design is changed)
+	var candidatesNumber int = 0
+	var islandsNumber int
+	var votesLayoutElect map[int][]int
+	votesSliceSquare := e.votes
+	candidatesNumber = len(e.islandsToVote)
+	islandsNumber = len(votesSliceSquare)
+	scoreInit := candidatesNumber + 1
+
+	//Transfer e.votes to a preference map with type "int"
+	for i := 0; i < islandsNumber-1; i++ {
+		for j := 0; j < candidatesNumber-1; j++ {
+			for k := 0; k < candidatesNumber-1; k++ {
```

Comment:
Why are we doing `candidatesNumber-1`? Won't this miss the last candidate?

Reply:
because index of the first candidate is 0

Reply:
Yes, it is, but you are also doing `<` as your condition which is non exclusive
```go
for i := 0; i < 5; i++
```
Will run 5 times, with `i = {0, 1, 2, 3, 4}`
Don't we want it to run `islandsNumber` times?

Reply:
Oh I'm sorry about this, as I didn't notice that when I modified the loop syntax, I'll modify it immediately.